### PR TITLE
Start changing the CDDL for mutually exclusive sdfChoice/enum

### LINKED ISSUE #87: Disallow using both `sdfChoice` and `enum` at the same time?
Revisiting the conversion from SDF to WoT, I was wondering how to deal with the edge case where both an `sdfChoice` and an `enum` are defined that contradict each other. For example, at the moment the following SDF model passes validation:

```json
{
  "sdfData": {
    "test": {
      "sdfChoice": {
        "foo": {
          "const": 1
        },
        "bar": {
          "const": 2
        }
      },
      "enum": [
        "foo",
        "bar"
      ]
    }
  }
}
```

Should it be valid to define a model like that? Or would it make sense to either define `enum` and `sdfChoice` as mutually exclusive in the schemas, or to add an assertion that prohibits using both in a set of data qualities?

(A model like this is probably a rare occurrence, however, I was wondering if a converter should raise an exception when encountering it or if, for example, `sdfChoice` should simply be preferred over `enum`.)

### PR DESCRIPTION
**Pull request by cabo**
Close #87

### CODE REVIEW

File: sdf-validation.jso.json [hunk 1]
```diff
@@ -1128,52 +549,427 @@
...
+        },
+        {
+          "type": "array",
+          "items": {
+            "type": "boolean"
+          }
+        },
         {
           "type": "object",
           "additionalProperties": {
```
Comment (akeranen) [inline sdf-validation.jso.json:975]: Empty additionalProperties on purpose?

**Review by akeranen — APPROVED** [review]
